tests/xdparser2/xdparser.mdclue.test.ts

```diff
@@ -40,139 +50,92 @@ it("handles bolding", () => {
 `)
 })
 
-it("handles italics and bolds", () => {
-  const xd = readFileSync("tests/xdparser2/inputs/alpha-bits.xd", "utf8")
-  const originalClue = "A1. Captain of the Pequod ~ AHAB"
-  const newMDClue = "A1. /Captain/ *of* the Pequod ~ AHAB"
-
-  expect(xdParser(xd.replace(originalClue, newMDClue)).clues.across[0]).toMatchInlineSnapshot(`
-{
-  "answer": "AHAB",
-  "body": "/Captain/ *of* the Pequod",
-  "metadata": undefined,
-  "number": 1,
-  "position": {
-    "col": 0,
-    "index": 0,
-  },
-  "tiles": [
-    {
-      "letter": "A",
-      "type": "letter",
-    },
-    {
-      "letter": "H",
-      "type": "letter",
-    },
-    {
-      "letter": "A",
-      "type": "letter",
-    },
-    {
-      "letter": "B",
-      "type": "letter",
-    },
+it("correctly handles non BBCode syntax", () => {
+  const newMDClue = "A1. The date of 2024/11/12. [MUAH]HEHE XD KISSES MEOW HEHE XD[/MUAH]"
+  const parsed = inlineMarkdownParser(newMDClue)
+  expect(parsed).toMatchInlineSnapshot(`
+[
+  [
+    "text",
+    "A1. The date of 2024/11/12. [MUAH]HEHE XD KISSES MEOW HEHE XD[/MUAH]",
   ],
-}
+]
 `)
-})
 
-it("handles URLs", () => {
-  const xd = readFileSync("tests/xdparser2/inputs/alpha-bits.xd", "utf8")
-  const originalClue = "A1. Captain of the Pequod ~ AHAB"
-  const newMDClue = "A1. [Captain](https://github.com/orta) **of** the ship /Pequod/ ~ AHAB"
+})
 
-  expect(xdParser(xd.replace(originalClue, newMDClue)).clues.across[0].bodyMD).toMatchInlineSnapshot(`
+it("handles links, bolds, italics, strikes, and dates", () => {
```

Comment:
this test is fine, but consider also adding individual tests for each tag types, and try to cover more edge cases

Reply:
ok